CHANGELOG.md

```diff
@@ -10,7 +10,7 @@ ENHANCEMENTS:
 
 * Bump the Ansible `community.general` collection to `4.6.1` and `community.docker` collection to `2.3.0`.
 * Add labels to loops in `tasks/config/template-config.yml` to reduce amount of output data.
-* Implement the `map`, `realip` and `split_clients` modules into the `http` core template.
+* Implemented the `gunzip`, `map`, `mirror`, `realip` and `split_clients` modules the `http` core template.
```

Comment:
Minor nitpick but let's keep the present imperative tense (Implement instead of Implemented)

Reply:
Okay, fixed